### ISSUE TITLE
About side

### ISSUE THREAD

**Issue by szgy66** [zjhthu/OANet#28]
Thank you for a great project. As I dig deeper into your code, I see that the run function of the NNMatcher(Learnedmatcher.py) function returns corr, sides, corr_Idx. Corr and corr_Idx are the corresponding match point and the index of descriptor respectively, but what does sides mean? Distribution of inside and outside points? Hope you can help answer, thank you

**Comment by zjhthu** [comment]
No, 'sides' means side information. Such as ratio test scores and whether the matches are mutual nearest neighbors.
